### DISCUSSION TITLE
建议使用umi max来创建项目还是使用隔壁家的antd pro？

### DISCUSSION THREAD

**Discussion by itzhoujun** [umijs/umi#11595] [General]
隔壁的antd pro底层也是用umi max，但是antd pro貌似做的更多。有点选择困难症，建议直接使用umi max创建项目还是用隔壁的antd pro呢？

**Comment by itzhoujun** [comment]
隔壁貌似不怎么更新了，umi还挺活跃

**Comment by fz6m** [comment]
ant pro 的模板用的是 max ，max 就是 umi 4 + 一堆内置插件，如果用 umi 就是不带任何插件，区别就在于有没有 [内置插件](https://umijs.org/docs/guides/use-plugins#%E4%BD%BF%E7%94%A8%E6%8F%92%E4%BB%B6) 而已，都是一个东西。

**Reply by itzhoujun** [reply]
我其实就是想问ant pro和umi max怎么选。

**Reply by fz6m** [reply]
ant pro 就是使用的 max ，所以只需要根据自己的需求来选择 umi 或者 max ，两者的区别在上文中已有 [相关描述](https://umijs.org/docs/guides/use-plugins#%E4%BD%BF%E7%94%A8%E6%8F%92%E4%BB%B6) 介绍，可以根据自己的需求选择。

**Reply by itzhoujun** [reply]
好的谢谢。
